app/templates/play/level.jade

```diff
@@ -1,3 +1,12 @@
+if view.isProduction() && !me.isPremium() && !view.isCourseMode() 
```

Comment:
`CocoView` already adds `isProduction` flag to the context, so we should either use that instead of `view.isProduction()` or put `isProduction: -> application.isProduction` in CocoView and not in specific subviews.